WebRoot/js/zimbraAdmin/accounts/view/ZaAccountMemberOfListView.js

```diff
@@ -750,21 +750,9 @@ ZaAccountMemberOfListView.prototype._setNoResultsHtml = function() {
 	var	div = document.createElement("div");
 	var msg = "";
 	if (this.getCurrentListId().indexOf(ZaAccount.A2_indirectMemberList) >= 0) {
-		var instance = this.parent && this.parent.getInstance();
-		if (instance && instance.memberOfLoaded) {
-			msg = ZaMsg.Account_Group_NoInDirectMember;
-		}
-		else {
-			msg = ZaMsg.MSG_HomeLoading;
-		}
+		msg = ZaMsg.Account_Group_NoInDirectMember;
 	}else if (this.getCurrentListId().indexOf(ZaAccount.A2_directMemberList) >= 0){
```

Comment:
I don't think this can be removed. The if else block we removed we inner blocks and not part of parent if-else ladder. Let me know if you meant something different?